tavern/internal/ent/schema/beacon.go

```diff
@@ -58,6 +58,14 @@ func (Beacon) Fields() []ent.Field {
 				entgql.Skip(entgql.SkipMutationUpdateInput),
 			).
 			Comment("Timestamp of when a task was last claimed or updated for the beacon."),
+		field.Uint64("interval").
+			Optional().
+			Annotations(
+				entgql.Type("Uint64"),
+				entgql.OrderField("INTERVAL"),
+				entgql.Skip(entgql.SkipMutationUpdateInput),
```

Comment:
I remember this being added to the other fields since they weren't being immutable.
This field will update. Do we still want to keep the Skip?